ubuntu_server_triage.py

```diff
@@ -53,13 +59,16 @@ def check_dates(start, end=None):
     return start, end
 
 
-def print_bugs(bugs):
+def print_bugs(bugs, open_in_browser=False):
     """
     Prints the bugs in a clean-ish format.
     """
     for bug in bugs:
         bug_url = 'https://bugs.launchpad.net/bugs/'
-        logging.info('%s%-7s - [%s] %s', bug_url, bug[0], bug[1], bug[2])
+        logging.info('%s%-7s - %-14s %-16s - %s',
+                     bug_url, bug[0], bug[3], ('[%s]' % bug[1]), bug[2])
```

Comment:
We still have to easily differ e.g. packages and stated with dashes or even spaces like "In Progress" so no boundary at all might not be that readable.
I looked at your list and chose this for my next push.

https://bugs.launchpad.net/bugs/1258320 (Triaged) [libvirt] Build libvirt with Hyper-V option